src/canvas-view/canvas-editor/toolbox-items.component.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+  'use strict';
+
+  angular.module('patternfly.canvas')
+    .component('toolboxItems', {
+      templateUrl: 'canvas-view/canvas-editor/toolbox-items.html',
+      bindings: {
+        items: '=',
+        startDragCallback: '=',
+        clickCallback: '=',
+        searchText: '='
+      },
```

Comment:
One-way binds for the ```callbacks``` using ```<``` do they work? Or do we need some form of the function/callback passed back out?

Reply:
The one-way binds seem to work in this case.

Reply:
@jeff-phillips-18 @dtaylor113 since the syntax on the consumption end should be the same is there a contrast opinion to updating the syntax?

Reply:
works for me.